packages/scalecube-microservice/src/api/MicroserviceOptions.ts

```diff
@@ -1,5 +1,11 @@
 import { Service } from '.';
 
+/**
+ * The options for the creation of microservice container
+ */
```

Comment:
/**
 * MicroserviceOptions - The options for the creation of microservice instance.
 *
 * @param `services` - An array of services
 *
 */